packages/wrangler/src/dev/inspect.ts

```diff
@@ -175,14 +175,13 @@ export function logConsoleMessage(
 	if (method in console) {
 		switch (method) {
 			case "dir":
-				console.dir(args);
+				logger.dir(args);
 				break;
 			case "table":
-				console.table(args);
+				logger.table(args.map((value) => ({ value })));
 				break;
 			default:
-				// eslint-disable-next-line prefer-spread
-				console[method].apply(console, args);
+				logger.log(...args);
```

Comment:
This is fine because they are logs from the worker which are informational and should be in stdout. Whereas stderr should contain wrangler errors/warnings

Reply:
But what about `assert`, `count`, `startGroup`, etc? This is not just about the stream it is written to.
I wonder if we are a bit stuffed from supporting these if we are no longer writing to the console directly and only writing to the stdout/stderr streams?

Reply:
Good point. No we're not stuffed, we still use the console methods – the calls just need to go via the logger so it can execute the before/after hooks

Reply:
https://github.com/cloudflare/workers-sdk/pull/6149/commits/e448c148a41db7d4f1cf3cd86f215e7124aabe0d